org.eclipse.xtext.testing/src/org/eclipse/xtext/testing/AbstractLanguageServerTest.xtend

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2016, 2020 TypeFox GmbH (http://www.typefox.io) and others.
+ * Copyright (c) 2016, 2020, 2021 TypeFox GmbH (http://www.typefox.io) and others.
```

Comment:
Please modify `2016, 2020, 2021` to `2016, 2021`.
From the [eclipse handbook](https://www.eclipse.org/projects/handbook/#ip-copyright-headers):

> The year is expressed as a single year or a range spanning the year of the very initial assertion and that of the most recent change.